code/modules/client/preferences.dm

```diff
@@ -1537,10 +1537,13 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 						preferred_map = maplist[pickedmap]
 
 				if ("clientfps")
-					var/desiredfps = input(user, "Choose your desired fps. (0 = synced with server tick rate (currently:[world.fps]))", "Character Preference", clientfps)  as null|num
+					var/desiredfps = input(user, "Please type in your Monitor Refresh Rate. Minimum Framerate is [world.fps])", "Character Preference", clientfps)  as null|num
```

Comment:
Minimum Framerate isn't actually 20. It's entirely possible to set it to any value between 1-19 currently. If someone wants to ruin their experience with that, let them.
I also disagree that the current message isn't informative and with sanitizing the value with something that isn't a constant.
Just add a line in the changelog that tell people to try out 60 fps.

In short, I liked your previous PR better and would happily slam the merge button on it.